spec/lib/que/job_spec.rb

```diff
@@ -176,4 +203,55 @@
       expect(job_class.default_attrs[:run_at]).to eq(run_at)
     end
   end
+
+  describe ".custom_log_context" do
```

Comment:
We already test the behaviour in the specs above so I'd just check the `ArgumentError`-case here.

Reply:
I just wanted to throw a bit more data at this method so I added the unit tests.
I cleaned it up a bit now and made it more relevant / less of a duplicate though.